### DISCUSSION TITLE
Fluent-Metro 1.4.1

### DISCUSSION THREAD

**Discussion by bonzibudd** [bonzibudd/Fluent-Metro#112] [General]
This is release 1.4.1 of Fluent-Metro. This update includes some minor tweaks and fixes. Here is the list of addressed issues:

 - #107 Submenu/arrow backgrounds
 - #109 "About" icon
 - #110 Programs button height
 - #111 Indented items

As always, please create a GitHub issue if you find any problems or unexpected behavior/changes. All feedback is appreciated, as it helps highlight changes to be considered.

<hr /><em>This discussion was created from the release <a href='https://github.com/bonzibudd/Fluent-Metro/releases/tag/v1.4.1'>Fluent-Metro 1.4.1</a>.</em>

**Comment by 333hronos** [comment]
Hello, what actually doing options "Horizontal / Vertical separators in sub-menu", "Curved menu / sub-menu selector", "Right-hand selector shape" ?

I've changed it on/off and see no changes...

p.s. Wonderful skin, great work!

**Reply by bonzibudd** [reply]
The horizontal separators option removes the horizontal lines in between some sub-menu items. For example, you can see this in the shutdown menu:

![image](https://user-images.githubusercontent.com/61938331/116000954-42ec0500-a5c0-11eb-9d70-627993e45c70.png)

When disabled:
![image](https://user-images.githubusercontent.com/61938331/116000972-56976b80-a5c0-11eb-84f8-3b04c58a1e11.png)

The vertical separators option does the same thing for vertical separators, but you will only see the difference when a list gets overflowed and splits into 2 columns.

**Reply by 333hronos** [reply]
Thanks!

And, what about options "Curved menu / sub-menu selector", and "Right-hand selector shape"?

**Reply by bonzibudd** [reply]
They determine if different selectors have rounded corners or not. The selectors are what appear when you hover over an item with your mouse. For the "Right-hand selector shape," it changes the right-hand selectors only, minus the shutdown button.
